packages/image_picker/image_picker_for_web/README.md

```diff
@@ -4,23 +4,9 @@ A web implementation of [`image_picker`][1].
 
 ## Limitations on the web platform
 
-Since Web Browsers don't offer direct access to their users' file system,
-this plugin provides a `PickedFile` abstraction to make access uniform
-across platforms.
+This plugin uses `XFile` objects to abstract files picked/created by the user.
 
-The web version of the plugin puts network-accessible URIs as the `path`
-in the returned `PickedFile`.
-
-### URL.createObjectURL()
-
-The `PickedFile` object in web is backed by [`URL.createObjectUrl` Web API](https://developer.mozilla.org/en-US/docs/Web/API/URL/createObjectURL),
-which is reasonably well supported across all browsers:
-
-![Data on support for the bloburls feature across the major browsers from caniuse.com](https://caniuse.bitsofco.de/image/bloburls.png)
-
-However, the returned `path` attribute of the `PickedFile` points to a `network` resource, and not a
-local path in your users' drive. See **Use the plugin** below for some examples on how to use this
-return value in a cross-platform way.
+Read more about `XFile` on the web in [`package:cross_file`'s README](https://pub.dev/packages/cross_file)
```

Comment:
(These README changes are all fine; we don't need to still README the deprecated implementation APIs.)

Reply:
> we don't need to still README the deprecated implementation APIs.

Having trouble parse that bit? Please rephrase?